### PR TITLE
Formatting cleanups and minor fixes.

### PR DESCRIPTION
**Pull request by mbrukman**
* Added missing code formatting where needed, e.g., to show `<` and `>`
* Capitalize uses of "Slack"
* Convert triple-backticks to single-backticks for inline code formatting
* Link directly to the license file
* Add a disclaimer section for visibility

### CODE REVIEW

**Comment by thesandlord** [comment]
One nit, why are the links to the files in code quotes? Makes it look strange IMO.

For example:
Edit [`constants.php`](constants.php)

vs:
Edit [constants.php](constants.php)

Aside from that LGTM

File: README.md [hunk 1]
```diff
@@ -25,20 +25,20 @@ With App Engine, you can run your slack inviter with the App Engine free tier an
...
-    * run ```composer install```
-* Edit [app.yaml](app.yaml)
-    * Replace ```<PROJECT-ID>``` with the project ID you just created
-* Edit [constants.php](constants.php)
-    * Replace ```<YOUR-SUBDOMAIN>``` with your team's slack subdomain (i.e If your slack is myteam.slack.com, your subdomain is 'myteam').
-    * Replace ```<YOUR-API-TOKEN>``` with your slack API token
+    * run `composer install`
+* Edit [`app.yaml`](app.yaml)
+    * Replace `<PROJECT-ID>` with the project ID you just created
+* Edit [`constants.php`](constants.php)
```
Comment (thesandlord) [inline README.md]: Why are there backticks around constants.php

File: README.md [hunk 2]
```diff
@@ -25,20 +25,20 @@ With App Engine, you can run your slack inviter with the App Engine free tier an
...
 * Visit [reCAPTCHA](https://www.google.com/recaptcha/admin), and register your website.
 * Install reCAPTCHA library with [Composer](https://getcomposer.org/)
-    * run ```composer install```
-* Edit [app.yaml](app.yaml)
-    * Replace ```<PROJECT-ID>``` with the project ID you just created
-* Edit [constants.php](constants.php)
-    * Replace ```<YOUR-SUBDOMAIN>``` with your team's slack subdomain (i.e If your slack is myteam.slack.com, your subdomain is 'myteam').
-    * Replace ```<YOUR-API-TOKEN>``` with your slack API token
+    * run `composer install`
+* Edit [`app.yaml`](app.yaml)
```
Comment (thesandlord) [inline README.md]: Why are there backticks around app.yaml

**Review by thesandlord — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mbrukman) [inline README.md]: Removed.

File: README.md (hunk 1 above)
Comment (mbrukman) [inline README.md]: Removed.

**Comment by mbrukman** [comment]
I used backticks on the filenames because filenames and paths get code-formatted similarly to shell commands and code samples, since they're used verbatim rather than serve as examples.

That said, I don't have a very strong feeling about it, so I've removed them in both cases.

**Review by thesandlord — APPROVED** [review]

**Pull request merged**